### ISSUE TITLE
[4.0.2][ES] Raspberry Pi 3, recalbox frozen trying to access System Settings

### ISSUE THREAD

**Issue by OMena82** [recalbox/recalbox-os#1229]
Fresh install. Dual Boot using Noobs (LibreElec + Recalbox). I tried to access system settings in order to use an usb drive but I cannot go further because recalbox is frozen. Rebooted, connected using SSH and obtained this:

/etc/init.d/S31emulationstation start
lvl0:         System "favorites" is missing name, path, extension, or command!
sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/etc/init.d/S31emulationstation: line 17:   615 Floating point exceptionHOME=/recalbox/share/system LC_ALL="en_US.UTF-8" SDL_VIDEO_GL_DRIVER=/usr/lib/libGLESv2.so SDL_NOMOUSE=1 /usr/bin/emulationstation

Hope you can help, thanks!

**Comment by OMena82** [comment]
I found a workaround. I used ssh to move the shared folder to my usb drive (using this tutorial [https://github.com/recalbox/recalbox-os/wiki/Auto-mount-of-usb-in-place-of-share-partition-(EN)](url))

I noticed that I didn't have the share partition in the fstab file so I didn't have to comment anything, I just added the line, rebooted and I was able to access System Settings. I changed the option in the settings so now everything is fine. I hope this may be helpful in order to find the bug :).
